sa/database.go

```diff
@@ -80,13 +71,33 @@ func NewDbMapFromConfig(config *mysql.Config, maxOpenConns int) (*gorp.DbMap, er
 		// In MariaDB, max_statement_time and long_query_time are both seconds.
 		// Note: in MySQL (which we don't use), max_statement_time is millis.
 		readTimeout := config.ReadTimeout.Seconds()
```

Comment:
Naming here is a bit confusing now, can we change this to `QueryTimeout` or something.